checkstyle/import-control.xml

```diff
@@ -177,6 +177,27 @@
     <allow pkg="org.apache.log4j" />
   </subpackage>
 
+  <subpackage name="trogdor">
+    <allow pkg="com.fasterxml.jackson" />
+    <allow pkg="javax.servlet" />
+    <allow pkg="javax.ws.rs" />
+    <allow pkg="net.sourceforge.argparse4j" />
+    <allow pkg="org.apache.kafka.clients.admin" />
+    <allow pkg="org.apache.kafka.clients.consumer" />
+    <allow pkg="org.apache.kafka.clients.producer" />
+    <allow pkg="org.apache.kafka.common"/>
```

Comment:
We'll probably end up using them eventually to communicate with Kafka.  But, I see your point-- there isn't any need to have them here now.